hibernate-core/src/main/java/org/hibernate/stat/internal/AbstractCacheableDataStatistics.java

```diff
@@ -12,14 +12,13 @@
 import org.hibernate.cache.spi.Region;
 import org.hibernate.stat.CacheableDataStatistics;
 
-import org.jboss.logging.Logger;
-
 /**
  * @author Steve Ebersole
  */
 public abstract class AbstractCacheableDataStatistics implements CacheableDataStatistics {
-	private static final Logger log = Logger.getLogger( AbstractCacheableDataStatistics.class );
-
+	// This magic number allow pre-5.3.7 releases RMI compatibility (which includes a JBoss Logger instance)	
+	private static final long serialVersionUID = 1417304055641145372L;
```

Comment:
To remove : "tricky tradeoff, we generally prefer to not define one as it's almost impossible to promise that it will be back/forward compatible in all JVMs."